flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/TemporalTableFunctionJoinITCase.scala

```diff
@@ -43,6 +43,62 @@ import scala.collection.mutable
 class TemporalTableFunctionJoinITCase(state: StateBackendMode)
   extends StreamingWithStateTestBase(state) {
 
+  @Test
+  def testProcessTimeInnerJoinUnionAll(): Unit = {
```

Comment:
could you move the case down to behind testProcessTimeInnerJoin, it would be better to have the basic test first, and then extended tests following.

Reply:
Done